backend/app/src/main/java/com/moh/phlat/backend/repository/ProcessDataFilterSpecification.java

```diff
@@ -0,0 +1,18 @@
+package com.moh.phlat.backend.repository;
+
+import java.util.List;
+
+import org.springframework.data.jpa.domain.Specification;
+
+import com.moh.phlat.backend.model.ProcessData;
+import com.moh.phlat.backend.model.SourceData;
+
+public interface ProcessDataFilterSpecification {
+	
+	public Specification<ProcessData> getDataWithMessages(Long controlId);
+	
+	public Specification<ProcessData> buildSpecificationWhereEqual(String columnKey, String value);
```

Comment:
Is it getting used anywhere?

Reply:
I think I was trying to keep some of the methods as close as possible between Source and Process.  This one has been removed.